### DISCUSSION TITLE
Is it possible to add a show or not configuration switch for the terminal tab's hover button?

### DISCUSSION THREAD

**Discussion by Alan3344** [Eugeny/tabby#9021] [General]
In some scenarios, I may need to open many tabs

At this point, when I left-click on the tab, a button with three dots of text appears, which makes it easy for me to left-click and the right-click menu pops up (I think that's what the right-click is supposed to do)
as a result, it's easy for me to accidentally click on this button and activate the menu subitem.

Expectation:
Without changing the original function, is it possible to provide a switch configuration item for whether the floating button is displayed?

![image](https://github.com/Eugeny/tabby/assets/60773912/a62199cf-cf84-4ac4-ad96-205ffe880c27)

![image](https://github.com/Eugeny/tabby/assets/60773912/386c08b2-88d4-4456-911a-7f99d95ec1cb)

**Comment by Eugeny** [comment]
You can disable the button in Settings -> Window -> Tabs. The context menu is still accessible via right click

**Comment by Alan3344** [comment]
Oh, I solved it

I found this option and turned it off. Clicking it will no longer expand the context menu.
Double-click to still edit the name

![image](https://github.com/Eugeny/tabby/assets/60773912/061cfe71-2998-4afc-8002-d71c67927f0f)
